### PR TITLE
Add bug report issue template

### PR DESCRIPTION
**Pull request by mrpollo**
Based on the current procedures and discussions with developers on the #qgroundcontrol channel on Slack

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 name: Feature request
```
Comment (hamishwillee) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Is it possible to have a feature request button that just goes to the discussion boards?

Comment (mrpollo) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Doesn't look like Github allows this, can you show me examples of issues created for feature requests? I wonder where they are coming from, regardless we need to make sure we make our channels official

Comment (mrpollo) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Perhaps we should remove the "Feature request" template, it's a misleading template

Comment (hamishwillee) [reply .github/ISSUE_TEMPLATE/feature_request.md]: I suspect we get more support requests via this than actual useful suggestions, but I don't monitor the channel. Ask Don :-)

**Comment by mrpollo** [comment]
Would be good to get this in sooner than later.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (DonLakeFlyer) [reply .github/ISSUE_TEMPLATE/feature_request.md]: > I suspect we get more support requests via this than actual useful suggestions

Correct. 50% is I'm trying to build QGC on some random platforms and it's not working. The rest is bug reports. Almost all of them with no enough information to do anyting about initially.

Feature Requests end up in Issues mostly from questions started on Forums where I asked the person to add a feature request in. Rarely do they start in Issues.

Comment (DonLakeFlyer) [reply .github/ISSUE_TEMPLATE/feature_request.md]: I think removing the Feature Request button completely makes a lot of sense.

**Comment by mrpollo** [comment]
Just sent a commit to remove the feature template, once this is in and CI is happy can we please merge @DonLakeFlyer?

**Comment by hamishwillee** [comment]
~~Ferris Bueler~~ @mrpollo, you're my hero :-)

**Pull request merged**